pkg/cluster/types.go

```diff
@@ -14,11 +14,11 @@ import (
 type PostgresRole string
 
 const (
-	// Master role
-	Master PostgresRole = "master"
-
-	// Replica role
-	Replica PostgresRole = "replica"
+	Master        PostgresRole = "master"
+	Replica       PostgresRole = "replica"
+	Leader        PostgresRole = "leader"
+	StandbyLeader PostgresRole = "standby_leader"
+	SyncStandby   PostgresRole = "sync_standby"
```

Comment:
I feel there is a comment missing here:
1. how does operator treat differently `Master`, `StandbyLeader` on the one hand and  `Leader` on the other ? Both `Master` and `StandbyLeader` are leaders. 
2. `SyncStanbdy` is still a `Replica`

While it is still possible to figure it out from the code, a short comment about why these are *not* completely independent states is worthy.

Reply:
`master` and `replica` are Spilo roles. The other ones role names returned by Patroni's `cluster` endpoint. Added comments